### PR TITLE
Clear TODOs and warnings

### PR DESCRIPTION
**Pull request by Ingloryon**

### CODE REVIEW

File: app/src/main/java/ch/epfl/polychef/recipe/RecipeStorage.java [hunk 1]
```diff
@@ -69,17 +88,19 @@ public void updateRecipe(Recipe recipe) {
...
-    public void readRecipeFromUuid(String uuid, CallHandler<Recipe> ch){
-        Preconditions.checkArgument(uuid != null, "Uuid should not be null");
-        Preconditions.checkArgument(ch != null, "Call handler should not be null");
+    /**
+     * Reads the Recipe with the given uuid.
+     * @param uuid the uuid of the recipe to read, must be non null
+     * @param ch the call handler of the recipe
+     */
+    public void readRecipeFromUuid(String uuid, @NonNull CallHandler<Recipe> ch){
+        Preconditions.checkArgument(!uuid.isEmpty(), "The given uuid must be non empty.");
```
Comment (Fukuyau) [inline app/src/main/java/ch/epfl/polychef/recipe/RecipeStorage.java:103]: Why did you delete a check on the fact that callhandler can't be null ?

**Review by Fukuyau — APPROVED** [review]
Very nice ! very good idea the Locale

**Review by wilderino — APPROVED** [review]
